eva-accession-import-automation/README.md

```diff
@@ -1,9 +1,10 @@
 # Pre-requisites
-* Install the **pandas** module in your local python environment
+* Install the **pandas** and **psycopg2** modules in your local python environment
     ```bash
-    pip install pandas --user
+    pip install -r requirements
```

Comment:
Is the .txt extension optional for requirements?

Reply:
oh, typo